scripts/detection/faster_rcnn/demo_faster_rcnn.py

```diff
@@ -8,14 +8,16 @@
 
 def parse_args():
     parser = argparse.ArgumentParser(description='Test with Faster RCNN networks.')
-    parser.add_argument('--network', type=str, default='faster_rcnn_resnet50_v1b_voc',
+    parser.add_argument('--network', type=str, default='faster_rcnn_resnet50_v1b_coco',
                         help="Faster RCNN full network name")
     parser.add_argument('--images', type=str, default='',
                         help='Test images, use comma to split multiple.')
-    parser.add_argument('--gpus', type=str, default='0',
+    parser.add_argument('--gpus', type=str, default='',
```

Comment:
Are `type=str` and `default=''` even necessary? Aren't those the defaults anyways?

Reply:
serves as extra guidance to user, doesn't hurt anyways.